src/app/app-platform/ContentApp.cpp

```diff
@@ -48,7 +48,7 @@ EmberAfStatus ContentApp::HandleReadAttribute(ClusterId clusterId, AttributeId a
                     "Read Attribute for endpoint " ChipLogFormatMEI " cluster " ChipLogFormatMEI " attribute " ChipLogFormatMEI,
                     ChipLogValueMEI(mEndpointId), ChipLogValueMEI(clusterId), ChipLogValueMEI(attributeId));
 
-    return EMBER_ZCL_STATUS_FAILURE;
+    return MATTER_CL_STATUS_FAILURE;
```

Comment:
Realistically... we should be replacing these with InteractionModel::Status.  Renaming is fine, but the name is not the major problem here.

Reply:
agreed. However replacing an error code with another one is a less trivial undertaking. Can we settle for a renaming in this PR and move the replacement of `EmberAFStatus` to `InteractionModel::Status` in a subsequent PR ?

Reply:
Sure.  Just not sure what the win is from this PR then, and how we justify the churn and potential API compat breaks....